lib/helpers.js

```diff
@@ -65,15 +65,41 @@ function getPolicy(group, resource) {
 
 function getRole(res, decoded, session) {
 
+  /**
+   * if role is attached to the session
+   * Return role
+   */
+
   if (decoded && decoded.role) {
     return decoded.role;
   }
+
+  /**
+   * if role is attached to the decoded
+   * Return role
+   */
+
   if (session && session.role) {
     return session.role;
   }
+
+  /**
+   * if role is not attached to the session or decoed object
+   * Return role
+   */
+
   return utils.deny(res, 404, 'REQUIRED: Role not found');
 }
 
+
+/**
+ * [resource finds the resource based of the baseurl specified]
+ * @param  {Function} next    [Express next function]
+ * @param  {[String}  url     [Request url]
+ * @param  {[string]}   baseUrl [The api baseUrl]
```

Comment:
Nit: Looks like you use "String" instead of "string" everywhere else, consider changing this to match. Extra 2 spaces here as well.